src/main/java/org/folio/marccat/dao/persistence/AuthorityModel.java

```diff
@@ -0,0 +1,31 @@
+package org.folio.marccat.dao.persistence;
+
+import org.folio.marccat.dao.AbstractDAO;
+import org.folio.marccat.dao.AuthorityModelDAO;
+
+/**
+ * @author elena
+ *
+ */
+public class AuthorityModel extends Model {
+
+  /**
+   * 
+   */
+  private static final long serialVersionUID = 1L;
+
+  /**
+   * Gets the authority model DAO.
+   *
+   * @return the authority model DAO
+   */
+  public AuthorityModelDAO getAuthorityModelDAO() {
+    return new AuthorityModelDAO();
+  }
+
+  @Deprecated
```

Comment:
Add the missing @deprecated Javadoc tag.

Reply:
I have added